README.md

```diff
@@ -160,7 +164,7 @@ Setup environment for CI to run tests. This is primarily designed to run on the
 ./script/cibuild
 ```
 
-### Database seed script
+### <a name="seeding"></a> Database seed script
```

Comment:
You can remove the HTML name bit here and instead link to the slug of this heading: `#database-seed-script`.